class/files/user/UserPages.php

```diff
@@ -171,7 +171,7 @@ private function getUserPagesFooter($moduleDirname, $tableName, $tableSoleName,
         $ret .= $uc->getUserMetaKeywords($moduleDirname);
         $ret .= $pc->getPhpCodeUnset('keywords');
         $ret .= $pc->getPhpCodeCommentLine('Description');
-        $ret .= $uc->getUserMetaDesc($moduleDirname, $language, $stuTableSoleName);
```

Comment:
e.g. table name is 'clients' , table sole name is 'client'
with `$ret .= $uc->getUserMetaDesc($moduleDirname, $language, $stuTableSoleName); ` you get `modulenameMetaDescription(_MA_MODULENAME_CLIENT_DESC);`
but in the language file you have created _MA_MODULENAME_CLIENTS_DESC

so, one of them must be changed